sw/airborne/subsystems/navigation/traffic_info.c

```diff
@@ -46,3 +49,45 @@ struct ac_info_ *get_ac_info(uint8_t id)
 {
   return &the_acs[the_acs_id[id]];
 }
+
+// 0 is reserved for ground station (id=0)
+// 1 is reserved for this AC (id=AC_ID)
+void SetAcInfo(uint8_t _id, float _utm_x /*m*/, float _utm_y /*m*/, float _course/*rad(CW)*/, float _alt/*m*/, float _gspeed/*m/s*/, float _climb, uint32_t _itow) {
+  if (acs_idx < NB_ACS) {
+    if (_id > 0 && the_acs_id[_id] == 0) {
+      the_acs_id[_id] = acs_idx++;
+      the_acs[the_acs_id[_id]].ac_id = _id;
+    }
+    the_acs[the_acs_id[_id]].east = _utm_x;// -  nav_utm_east0;
+    the_acs[the_acs_id[_id]].north = _utm_y;// - nav_utm_north0;
+    the_acs[the_acs_id[_id]].course = _course;
+    the_acs[the_acs_id[_id]].alt = _alt;
+    the_acs[the_acs_id[_id]].gspeed = _gspeed;
+    the_acs[the_acs_id[_id]].climb = _climb;
+    the_acs[the_acs_id[_id]].itow = _itow;
+  }
+}
+
+// 0 is reserved for ground station (id=0)
+// 1 is reserved for this AC (id=AC_ID)
+void SetAcInfoLLA(uint8_t id, uint32_t lat, uint32_t lon, uint32_t alt, uint32_t course, uint16_t gspeed, uint16_t climb,
+               uint32_t tow)
+{
+  if (acs_idx < NB_ACS) {
+    if (id > 0 && the_acs_id[id] == 0) {
+      the_acs_id[id] = acs_idx++;
+      the_acs[the_acs_id[id]].ac_id = id;
+    }
+
+    struct EcefCoor_i ecef_pos;
+    struct LlaCoor_i lla = {.lat = lat, .lon = lon, .alt = alt};
+    ecef_of_lla_i(&ecef_pos, &lla);
+    the_acs[the_acs_id[id]].east = (float)ecef_pos.x * 100;
```

Comment:
`ecef.x` is clearly not an `east` value, same for other parameters. This could be confusing in a future use. The structure should be probably extended to use proper position and speed formats.